modules/virt-accessing-rdp-console.adoc

```diff
@@ -6,71 +6,101 @@
 [id="virt-accessing-rdp-console_{context}"]
 = Connecting to a Windows virtual machine with an RDP console
 
-The Remote Desktop Protocol (RDP) provides a better console experience for
-connecting to Windows virtual machines.
-
-To connect to a Windows virtual machine with RDP, specify the IP address of the
-attached L2 NIC to your RDP client.
+Create a Kubernetes `Service` object to connect to a Windows virtual machine (VM) by using your local Remote Desktop Protocol (RDP) client.
 
 .Prerequisites
 
-* A running Windows virtual machine with the QEMU guest agent installed. The
-`qemu-guest-agent` is included in the VirtIO drivers.
-* A layer 2 NIC attached to the virtual machine. 
-* An RDP client installed on a machine on the same network as the
-Windows virtual machine.
+* A running Windows virtual machine with the QEMU guest agent installed. The `qemu-guest-agent` is included in the VirtIO drivers.
+* An RDP client installed on a machine on the same network as the Windows VM.
 
 .Procedure
 
-. Log in to the {VirtProductName} cluster through the `oc` CLI tool as a user with
-an access token.
+. Edit the `VirtualMachine` manifest to add the label for service creation:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm-ephemeral
+  namespace: example-namespace
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        special: key <1>
+# ...
+----
+<1> Add the label `special: key` in the `spec.template.metadata.labels` section.
++
+
+[NOTE]
+====
+Labels on a virtual machine are passed through to the pod. The `special: key` label must match the label in the `spec.selector` attribute of the `Service` manifest.
+====
+
+. Save the `VirtualMachine` manifest file to apply your changes.
+. Create a `Service` manifest to expose the VM:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: vmservice <1>
+  namespace: example-namespace <2>
+spec:
+  ports:
+  - targetPort: 3389 <3>
+    protocol: TCP
+    port: 27017
```

Comment:
```suggestion
  - port: 3389 <3>
    protocol: TCP
```

There is no benefit of using both `port` and `targetPort` here. We can simplify this to just `port` and rely on the defaulting.